File: Chain of Responsibility Pattern/Patient.java
package patient;

import diseaseseverity.DiseaseSeverity;

public class Patient{
	
	private String name;
	private DiseaseSeverity diseaseSeverity;
	private int age;
	
	public Patient(String name, DiseaseSeverity diseaseSeverity, int age){
		this.name = name;
		this.diseaseSeverity = diseaseSeverity;
		this.age = age;	
	}
	
	public String getName(){
		return this.name;
	}
	
	public DiseaseSeverity getDiseaseSeverity(){
		return this.diseaseSeverity;
	}
	
	public int getAge(){
		return this.age;
	}
		
}

File: Design Patterns/Chain of Responsibility Pattern in JavaScript/Patient.js
class Patient {
    #name;
    #diseaseSeverity;
    #age;

    constructor(name, diseaseSeverity, age) {
        this.#name = name;
        this.#diseaseSeverity = diseaseSeverity;
        this.#age = age;
    }

    get name() {
        return this.#name;
    }

    get diseaseSeverity() {
        return this.#diseaseSeverity;
    }

    get age() {
        return this.#age;
    }

}

module.exports = Patient;